core/Pos/Binary/Core/Common.hs

```diff
@@ -0,0 +1,76 @@
+module Pos.Binary.Core.Common () where
+
+import           Universum
+
+import           Pos.Binary.Class (Bi (..), Cons (..), Field (..), deriveSimpleBi)
+import           Pos.Core.Common.Types (Coin, mkCoin, unsafeGetCoin)
+import qualified Pos.Core.Common.Types as T
+import qualified Pos.Data.Attributes as A
+import           Pos.Util.Orphans ()
+
+-- kind of boilerplate, but anyway that's what it was made for --
+-- verbosity and clarity
+
+instance Bi (A.Attributes ()) where
+    encode = A.encodeAttributes []
+    decode = A.decodeAttributes () $ \_ _ _ -> pure Nothing
+
+instance Bi T.CoinPortion where
+    encode = encode . T.getCoinPortion
+    decode =
+        T.mkCoinPortion <$> (decode @Word64) >>= \case
+            Left err          -> fail err
+            Right coinPortion -> return coinPortion
+
+instance Bi T.BlockCount where
+    encode = encode . T.getBlockCount
+    decode = T.BlockCount <$> decode
+
+-- serialized as vector of TxInWitness
+--instance Bi T.TxWitness where
```

Comment:
Do you think it's worthwhile keeping this comment, if this is genuinely-unused code?

Reply:
Good catch, it's indeed obsolete (was obsolete even before my PR), I will delete this comment.